scripts/generate-docs.sh

```diff
@@ -27,8 +24,56 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+echo "Removing build files..."
+git clean -fx
+if [ $? -ne 0 ]; then
+  echo -e "\033[0;31m" 1>&2 # Red
+  echo "Could not remove untracked/ignored files"
+  echo -e "\033[0m" 1>&2 # Normal Color
+  exit 1
+fi
+
+echo "Main \`npm install\`..."
+npm install
+if [ $? -ne 0 ]; then
+  echo -e "\033[0;31m" 1>&2 # Red
+  echo "\`npm install\` failed"
+  echo -e "\033[0m" 1>&2 # Normal Color
+  exit 1
+fi
+
+echo "Getting types for es6 promises..."
+npm install @types/es6-promise
+if [ $? -ne 0 ]; then
+  echo -e "\033[0;31m" 1>&2 # Red
+  echo "Couldn't get types for es6 promises"
+  echo -e "\033[0m" 1>&2 # Normal Color
+  exit 1
+fi
+
+echo "Compiling down to es5..."
+node node_modules/typescript/bin/tsc --target es5
+if [ $? -ne 0 ]; then
+  echo -e "\033[0;31m" 1>&2 # Red
+  echo "Couldn't compile for es5"
+  echo -e "\033[0m" 1>&2 # Normal Color
+  exit 1
+fi
+
+# Remove unneeded type
+npm remove @types/es6-promise
+
+echo "Installing the testapp..."
+npm run install_testapp
+if [ $? -ne 0 ]; then
+  echo -e "\033[0;31m" 1>&2 # Red
+  echo "Couldn't install testapp"
+  echo -e "\033[0m" 1>&2 # Normal Color
+  exit 1
+fi
 
-# Generate files
+echo "Building the website..."
+cd website
 npm run build
```

Comment:
Sorry about that.  Looks like my clean from a couple lines up wasn't as thorough as I though, so I didn't catch this.  Fixed.